seqr/views/apis/anvil_workspace_api.py

```diff
@@ -110,6 +110,13 @@ def create_project_from_workspace(request, namespace, name):
     if errors:
         return create_json_response({'errors': errors}, status=400)
 
+    # Validate the VCF to see if it contains all the required samples
+    samples = get_vcf_samples(data_path)
+    missing_samples = [record['individualId'] for record in pedigree_records if record['individualId'] not in samples]
+    if missing_samples:
+        return create_json_response(
+            {'errors': 'There are missing samples in the data: {}'.format(', '.join(missing_samples))}, status=400)
```

Comment:
A better error would be "The following samples are included in the pedigree file but are missing from the VCF: {}"